user-console/src/main/resources-filtered/org/pentaho/mantle/public/browser/index.jsp

```diff
@@ -53,13 +53,21 @@
   }
 </script>
 
-<!-- libs -->
-<script type="text/javascript" src="lib/jquery/jquery-${jquery.version}.js"></script>
-<script type="text/javascript" src="lib/underscore/underscore-min.js"></script>
-<script type="text/javascript" src="lib/backbone/backbone.js"></script>
-
 <!-- Require File Browser -->
 <script type="text/javascript">
+
+  require.config({
```

Comment:
Inline RequireJS configuration to complement the config already loaded in `require` by `webcontext.js`. This is a local configuration that does not affect other parts/contexts of the platform.